CITATION.cff

```diff
@@ -0,0 +1,20 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+type: "software"
+title: "mamba"
+doi: "10.5281/zenodo.7434205"
```

Comment:
DOI goes 404?

https://doi.org/10.5281/zenodo.7434205

Also not on Zenodo: https://zenodo.org/records/7434205